config/conf.json

```diff
@@ -144,6 +144,22 @@
                 "include_in_custom_gce" : true
             }
         },
+        {
+            "name" : "terra-aou-bioconductor-tidyverse-3.16.0",
```

Comment:
Those names is a bit confusing. I saw 
`terra-aou-bioconductor-tidyverse-3.16.0` 
`aou-rstudio-bioconductor`
and `aou-rstudio-bioconductor-3.16.0`
I think the naming conversion is dir name is the same as image name. Suggest to make them be consistent with other images. 
Also, suggest to remove 3.16.0 from image name as it is indicated in version.

Reply:
Yeah, the image dir should not have a version, nor should this name. Versioning is handled elsewhere. Also recommend not including any versioning in the dockerfile for the same reason

Reply:
Updated names